pkgs/stdenv/generic/setup.sh

```diff
@@ -3,6 +3,9 @@ set -o pipefail
 
 : ${outputs:=out}
 
+# If unset, assume the default hardening flags.
+: ${NIX_HARDENING_ENABLE="fortify stackprotector pic strictoverflow format relro bindnow"}
```

Comment:
It seems that `=` instead of `:=` works, but is it documented?

Reply:
I could do that; do I then need to do something similar for bintools, though?

Reply:
Yeah if we go with two separate env vars, then it is not a problem.